src/token/fedauth-info-parser.js

```diff
@@ -0,0 +1,62 @@
+const async = require('async');
+
+var readFedAuthInfoOpt = (data, infoData, callback) => {
+  const FEDAUTHINFOID = {
```

Comment:
What do you think about pulling this constant to the top level scope, to ensure it does not get re-defined on each function call?